src/scene/materials/standard-material.js

```diff
@@ -183,12 +183,42 @@ let _params = new Set();
  * @property {number} clearCoatBumpiness The bumpiness of the clear coat layer. This value scales
  * the assigned main clear coat normal map. It should be normally between 0 (no bump mapping) and 1
  * (full bump mapping), but can be set to e.g. 2 to give even more pronounced bump effect.
+ * @property {boolean} useIridescence Enable thin-film iridescence.
+ * @property {Texture|null} iridescenceMap The per-pixel iridescence intensity. Only used when
+ * useIridescence is enabled.
+ * @property {number} iridescenceMapUv Iridescence map UV channel.
+ * @property {Vec2} iridescenceMapTiling Controls the 2D tiling of the iridescence map.
+ * @property {Vec2} iridescenceMapOffset Controls the 2D offset of the iridescence map. Each component is
+ * between 0 and 1.
+ * @property {number} iridescenceMapRotation Controls the 2D rotation (in degrees) of the iridescence
+ * map.
+ * @property {string} iridescenceMapChannel Color channels of the iridescence map to use. Can be "r",
+ * "g", "b" or "a".
+ * @property {Texture|null} iridescenceThicknessMap The per-pixel iridescence thickness. Defines a
+ * gradient weight between iridescenceThicknessMin and iridescenceThicknessMax. Only used when
+ * useIridescence is enabled.
+ * @property {number} iridescenceThicknessMapUv Iridescence thickness map UV channel.
+ * @property {Vec2} iridescenceThicknessMapTiling Controls the 2D tiling of the iridescence
+ * thickness map.
+ * @property {Vec2} iridescenceThicknessMapOffset Controls the 2D offset of the iridescence
+ * thickness map. Each component is between 0 and 1.
+ * @property {number} iridescenceThicknessMapRotation Controls the 2D rotation (in degrees)
+ * of the iridescence map.
+ * @property {string} iridescenceThicknessMapChannel Color channels of the iridescence thickness
+ * map to use. Can be "r", "g", "b" or "a".
+ * @property {number} iridescenceThicknessMin The minimum thickness for the iridescence layer.
+ * Only used when an iridescence thickness map is used.
+ * @property {number} iridescenceThicknessMax The maximum thickness for the iridescence layer.
+ * Used as the 'base' thickness when no iridescence thickness map is defined.
+ * @property {number} iridescenceRefractionIndex The index of refraction of the iridescent
+ * thin-film.
```

Comment:
Worth specifying a range, or giving guidance as to the effect of different values?

Reply:
It's a bit tricky to make it easy to understand. The index of refraction will change the phase shift of the iridescent effect, as in how frequent the different color bands will appear on the surface. Reasonable values range from between 0 to ~7 where the upper limit represents the highest measured index of refraction of a physical surface to date.

In short, it's very technical 🙂 .

Reply:
Added documentation links to the glTF extension spec which explains how the iridescence refraction index affects color phase shift of the material.